deploy-vm.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+ 
+# Install requirements (docker)
+
+echo "Step 1"
```

Comment:
@yashdusing it would be a good idea to add more descriptive statements instead of just `Step 1` `Step 2`. You can use statements like `Pulling docker image...`, etc